interchain-exchange/tutorial/09-walkthrough.md

```diff
@@ -29,7 +29,17 @@ The ports for `Mars` are default ports, while for `Venus` the port details can b
 
 ```bash
 # relayer configuration
-starport relayer configure -a --source-rpc "http://0.0.0.0:26657" --source-faucet "http://0.0.0.0:4500" --source-port "ibcdex" --source-version "ibcdex-1" --target-rpc "http://0.0.0.0:26659" --target-faucet "http://0.0.0.0:4501" --target-port "ibcdex" --target-version "ibcdex-1"
+starport relayer configure -a \
+--source-rpc "http://0.0.0.0:26657" \
+--source-faucet "http://0.0.0.0:4500" \
+--source-port "ibcdex" \
+--source-version "ibcdex-1" \
+--source-gasprice "0.0000025token" \
```

Comment:
`--source-gasprice` this option is not available yet on the current version of Starport, it will actually be for Starport 0.16. Therefore, I would say we can create another branch for this change to merge once the new version of Starport is released

Reply:
got it. In the meantime I'll add instructions about which value for gas to set when prompted

Reply:
We can keep using this flag to have the command fully automatized. We should just keep the tutorial always synchronized with the latest release of Starport.

We have a branch for the changes for the next release of Starport: https://github.com/cosmos/sdk-tutorials/pull/600 I can add this change here